### PR TITLE
USDC swap banner

### PR DESCRIPTION
**Pull request by MeanBoyCousin**
# Task: RYSK-237

## Description

- Adds a banner to direct people to swap their bridged USDC.e to native USDC.
- Fixes a few small styling issues.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update
- [ ] Quality of life improvement
- [ ] Package upgrades

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have NATSPEC'd any new functions
- [x] If appropriate, I have properly tested my new functionality

### CODE REVIEW

File: packages/front-end/src/components/Header.tsx [hunk 1]
```diff
@@ -13,53 +14,70 @@ const links = [
...
+              {`Rysk Finance is using native USDC. To swap your bridged USDC.e, you can `}
+              <a
+                className="text-cyan-dark-compliant underline"
+                href="https://jumper.exchange/?fromChain=42161&fromToken=0xff970a61a04b1ca14834a43f5de4533ebddb5cc8&toChain=42161&toToken=0xaf88d065e77c8cc2239327c5edb3a432268e5831"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                {`click here.`}
+              </a>
+            </span>
```
Comment (thundering-silence) [inline packages/front-end/src/components/Header.tsx:73]: Copy assumes people know the difference between native and USDC.e, is this the case? If not - might be worth being a little more verbose?

**Review by thundering-silence — APPROVED** [review]
Just a small concern over copy - update only if you think it's worth it else merge 👍

File: packages/front-end/src/components/Header.tsx (hunk 1 above)
Comment (MeanBoyCousin) [reply packages/front-end/src/components/Header.tsx:73]: True, we do at least mention `native` and `bridged` though. @ugolino are you happy with copy or do you think we need to add more detail?

Comment (ugolino) [reply packages/front-end/src/components/Header.tsx:73]: good point. how about we add official link? 

https://www.circle.com/blog/arbitrum-usdc-now-available

**Review by thundering-silence — DISMISSED** [review]
✅

**Pull request merged**